packages/sra-report/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const utils = {
+    log(...args: any[]): void {
+        console.log(...args); // tslint:disable-line:no-console
```

Comment:
we have a couple instances of this pattern across multiple packages, we should consider moving it to the utils package or something

Reply:
Yeah, I'm 100% for this.

Reply:
I'll follow up with a separate PR